dash-renderer/src/components/error/FrontEnd/FrontEndError.css

```diff
@@ -2,6 +2,22 @@
     margin-top: 10px;
 }
 
+.dash-fe-error__icon-x {
+    position: absolute;
+    right: 0;
+    top: 0;
+    color: #ccc;
+    font-size: 20px;
+    cursor: pointer;
+    margin-right: 5px
+}
+
+.dash-fe-error__icon-x:hover
+{
+ color:black;
```

Comment:
That's probably a stronger effect than we want - really we just need enough to be able to see a change in the X. Closest analog I see is the grey debug menu buttons, which starts out about the same as `#ccc` (just a tiny bit tilted toward blue)

https://github.com/plotly/dash/blob/e7f4984d0b58511cdd8d4c97ac5ecf963dd89c35/dash-renderer/src/components/error/menu/DebugMenu.css#L70

And on hover it gets just a bit darker

https://github.com/plotly/dash/blob/e7f4984d0b58511cdd8d4c97ac5ecf963dd89c35/dash-renderer/src/components/error/menu/DebugMenu.css#L84

I'm not sure where those exact colors came from, but we may be able to just copy them both.

Reply:
I see, I'll make the change.  Thanks!